### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by iAmDusan**
just readme changes

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ app.use('/api/finances', financeRouter);
 
 
 app.get('/', (req, res) => {
-  res.send('Hello, from Lucas deployment!');
+  res.send('Hello, world!');
```
Comment (RobertWiggins) [inline src/app.js]: Should we take this out now since we are close to production or no?

Comment (michaelbravo777) [reply src/app.js]: yes.  it makes us look like noobs.

File: README.md [hunk 2]
```diff
@@ -20,6 +20,315 @@ env MIGRATION_DB_NAME=stay-informed-test npm run migrate
...
+#### Civics API
+
+`GET https://www.googleapis.com/civicinfo/v2/representatives??key=${config.CIVIC_API_KEY}&address=${address}`
+
+This returns a user's state, congressional district, and other data associated with this registered address.
+
+For our needs, we take the state and congressional district data and use them in our request to ProPublica and Open Secrets
+
+```
+{
```
Comment (RobertWiggins) [inline README.md]: I think the spacing/indentation is off here

File: README.md [hunk 3]
```diff
@@ -20,6 +20,315 @@ env MIGRATION_DB_NAME=stay-informed-test npm run migrate
...
+  },
+  "ocd-division/country:us/state:ny/county:new_york": {
+   "name": "New York County",
+   "alsoKnownAs": [
+    "ocd-division/country:us/state:ny/borough:manhattan",
+    "ocd-division/country:us/state:ny/place:new_york/county:new_york"
+   ],
+   "officeIndices": [
+    11,
+    12
```
Comment (RobertWiggins) [inline README.md]: I think the spacing/indentation is off here

Comment (lucasvocos) [reply README.md]: Was just a copy/paste from the response in Postman

**Pull request merged**